conf/config.inc.php

```diff
@@ -174,6 +174,9 @@
 # What attribute should be changed by the changesshkey action?
 $change_sshkey_attribute = "sshPublicKey";
 
+# What objectClass is required for that attribute?
+$change_sshkey_objectClass = "";
```

Comment:
I think we should set the default value to ldapPublicKey, as this is the standard objectClass.

The only impact would be for people using the attribute with another object class, but this should be seldom.